src/FinderCommand.php

```diff
@@ -3,8 +3,11 @@
 namespace PhpUnitFinder;
 
 use PHPUnit\Framework\TestCase;
-use PHPUnit\TextUI\TestSuiteMapper;
+use PHPUnit\Runner\Version;
+use PHPUnit\TextUI\CliArguments\Builder;
+use PHPUnit\TextUI\Configuration\Registry;
 use PHPUnit\TextUI\XmlConfiguration\Loader;
+use PHPUnit\TextUI\XmlConfiguration\TestSuiteMapper;
```

Comment:
This doesn't actually work on phpunit9 since the class moved
```
skpr@b1cfcf46df40:data $ ./bin/phpunit-finder unit kernel | wc -l
PHP Fatal error:  Uncaught Error: Class "PHPUnit\TextUI\XmlConfiguration\TestSuiteMapper" not found in /data/vendor/previousnext/phpunit-finder/src/FinderCommand.php:56
Stack trace:
```

Can we just spin another major and remove the double version handling?

Reply:
Yep sure